clang/test/Driver/riscv-arch.c

```diff
@@ -365,24 +365,30 @@
 // RUN: -fsyntax-only 2>&1 | FileCheck -check-prefix=RV32-ZFHMIN %s
 // RV32-ZFHMIN: "-target-feature" "+zfhmin"
 
-// RUN: not %clang --target=riscv32-unknown-elf -march=rv32iztso -### %s \
+// RUN: not %clang --target=riscv32-unknown-elf -march=rv32izalasr -### %s \
```

Comment:
Does this mean we should change this when `Zalasr` is no longer experimental?
(Thought I think we don't have a way to avoid this, we will always need an experimental extension here)

Reply:
Yeah.